--- origsrc/GeoIP-1.4.4/conf/Makefile.am	2002-11-21 00:12:37.000000000 -0600
+++ src/GeoIP-1.4.4/conf/Makefile.am	2008-03-17 20:48:55.687500000 -0500
@@ -8,6 +8,6 @@
 	@if test -f $(DESTDIR)$(DEFAULT_CONFIG_FILE) ; then \
 	        echo "$@ will not overwrite existing $(DESTDIR)$(DEFAULT_CONFIG_FILE)" ; \
 	else \
-	        echo "$(INSTALL_DATA) GeoIP.conf.default $(DESTDIR)$(DEFAULT_CONFIG_FILE)"; \
-	        $(INSTALL_DATA) GeoIP.conf.default $(DESTDIR)$(DEFAULT_CONFIG_FILE); \
+	        echo "$(INSTALL_DATA) $(srcdir)/GeoIP.conf.default $(DESTDIR)$(DEFAULT_CONFIG_FILE)"; \
+	        $(INSTALL_DATA) $(srcdir)/GeoIP.conf.default $(DESTDIR)$(DEFAULT_CONFIG_FILE); \
 	fi
--- origsrc/GeoIP-1.4.4/data/Makefile.am	2002-11-21 00:12:37.000000000 -0600
+++ src/GeoIP-1.4.4/data/Makefile.am	2008-03-17 20:48:55.687500000 -0500
@@ -8,6 +8,6 @@
 	@if test -f $(DESTDIR)$(DEFAULT_DB_FILE) ; then \
 	        echo "$@ will not overwrite existing $(DESTDIR)$(DEFAULT_DB_FILE)" ; \
 	else \
-	        echo "$(INSTALL_DATA) GeoIP.dat $(DESTDIR)$(DEFAULT_DB_FILE)"; \
-	        $(INSTALL_DATA) GeoIP.dat $(DESTDIR)$(DEFAULT_DB_FILE); \
+	        echo "$(INSTALL_DATA) $(srcdir)/GeoIP.dat $(DESTDIR)$(DEFAULT_DB_FILE)"; \
+	        $(INSTALL_DATA) $(srcdir)/GeoIP.dat $(DESTDIR)$(DEFAULT_DB_FILE); \
 	fi
--- origsrc/GeoIP-1.4.4/man/Makefile.am	2007-02-08 17:20:27.000000000 -0600
+++ src/GeoIP-1.4.4/man/Makefile.am	2008-03-17 20:48:56.281250000 -0500
@@ -6,5 +6,5 @@
 LOOKUP_MAN = $(mandir)/man1/geoiplookup.1
 
 install-data-hook: install-man
-	cat geoipupdate.1 | sed s,DATADIR,$(pkgdatadir), | sed s,CONF_DIR,$(sysconfdir), > $(DESTDIR)$(UPDATE_MAN)
-	cat geoiplookup.1 | sed s,DATADIR,$(pkgdatadir), > $(DESTDIR)$(LOOKUP_MAN)
+	cat $(srcdir)/geoipupdate.1 | sed s,DATADIR,$(pkgdatadir), | sed s,CONF_DIR,$(sysconfdir), > $(DESTDIR)$(UPDATE_MAN)
+	cat $(srcdir)/geoiplookup.1 | sed s,DATADIR,$(pkgdatadir), > $(DESTDIR)$(LOOKUP_MAN)
